kustomize/base/wf_crd/wf_crd.yaml

```diff
@@ -7,7 +7,23 @@ spec:
   # group name to use for REST API: /apis/<group>/<version>
   group: flyte.lyft.com
   # version name to use for REST API: /apis/<group>/<version>
-  version: v1alpha1
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          properties:
+            spec:
+              type: object
+              properties:
+                cronSpec:
+                  type: string
+                image:
+                  type: string
+                replicas:
+                  type: integer
   # either Namespaced or Cluster
```

Comment:
This worked for me

```suggestion
  versions:
  - name: v1alpha1
    schema:
      openAPIV3Schema:
        description: 'FlyteWorkflow: represents one Execution Workflow object'
        type: object
    served: true
    storage: true
  # either Namespaced or Cluster
```

Reply:
thanks @EngHabu , i tested in my laptop, and it worked.

Reply:
Thank you for confirming. Can you please update the PR so I can approve and merge?

Reply:
updated the changes, thanks again